examples/client.rs

```diff
@@ -107,18 +109,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         let mut stream = conn.send_request(req).await?;
         stream.finish().await?;
 
-        eprintln!("Receiving response ...");
-        let resp = stream.recv_response().await?;
-
-        eprintln!("Response: {:?} {}", resp.version(), resp.status());
-        eprintln!("Headers: {:#?}", resp.headers());
-
-        while let Some(chunk) = stream.recv_data().await? {
-            let mut out = tokio::io::stdout();
-            out.write_all(&chunk).await?;
-            out.flush().await?;
-        }
-        Ok::<(), Box<dyn std::error::Error>>(())
+        Ok::<_, Box<dyn std::error::Error>>(handle_request(stream).await)
```

Comment:
I think this makes it a `Result<Result<(), E>, E>`.

Reply:
If rustc says this compiles it means everything is all right isn't it :smile:. I remove the function.
Thanks.